gnomad_qc/v4/create_release/create_release_sites_ht_genomes.py

```diff
@@ -0,0 +1,790 @@
+    # and update the keys to the path for any new tables
+
+    included_datasets = {}
+    if "release" in tables:
+        with hl.utils.hadoop_open(
+            included_datasets_json_path(
+                test=test,
+                release_version=hl.eval(get_config()["release"]["ht"].version),
+            )
+        ) as f:
+            included_datasets = json.loads(f.read())
+
+    included_datasets.update(
+        {t: get_config(release_exists=release_exists)[t]["path"] for t in tables}
+    )
+    with hl.utils.hadoop_open(
+        included_datasets_json_path(test=test, release_version=args.version), "w"
+    ) as f:
+        f.write(hl.eval(hl.json(included_datasets)))
+
+    return joined_ht
+
+
+def main(args):
+    """Create release ht."""
+    hl.init(
+        log="/create_release_ht_genomes.log",
+        tmp_dir="gs://gnomad-tmp-4day",
+        default_reference="GRCh38",
+    )
+
+    logger.info("Creating genomes release HT...")
+    ht = join_hts(
+        args.base_table,
+        args.tables_for_join,
+        args.new_partition_percent,
+        args.test,
+        args.release_exists,
+    )
+
+    # Filter out chrM, AS_lowqual sites (these sites are dropped in the final_filters HT
+    # so will not have information in `filters`) and AC_raw == 0.
+    ht = hl.filter_intervals(ht, [hl.parse_locus_interval("chrM")], keep=False)
+    ht = ht.filter(hl.is_defined(ht.filters) & (ht.freq[1].AC > 0))
+
+    ht = ht.select_globals(**get_select_global_fields(ht))
+
+    # Add additional globals that were not present on the joined HTs.
+    ht = ht.annotate_globals(
+        filtering_model=ht.filtering_model.drop("model_id"),
+        vep_globals=ht.vep_globals.annotate(
+            gencode_version=GENCODE_VERSION,
+            mane_select_version=MANE_SELECT_VERSION,
+        ),
+        tool_versions=ht.tool_versions.annotate(
+            dbsnp_version=DBSNP_VERSION,
+            sift_version=SIFT_VERSION,
+            polyphen_version=POLYPHEN_VERSION,
+        ),
+        vrs_versions=hl.struct(
+            **{
+                "vrs_schema_version": VRS_SCHEMA_VERSION,
+                "vrs_python_version": VRS_PYTHON_VERSION,
+                "seqrepo_version": SEQREPO_VERSION,
+            },
+        ),
+        date=datetime.now().isoformat(),
+        version=args.version,
+    )
+
+    # Reorder fields to match final schema.
+    ht = ht.select(*FINALIZED_SCHEMA["rows"]).select_globals(
+        *FINALIZED_SCHEMA["globals"]
+    )
+
+    output_path = (
+        f"{qc_temp_prefix()}release/gnomad.genomes.sites.test.updated_101723.ht"
+        if args.test
+        else release_sites(data_type="genomes").path
+    )
+    logger.info("Writing out release HT to %s", output_path)
+    ht = ht.checkpoint(
+        output_path,
+        args.overwrite,
+    )
+
+    logger.info("Final release HT schema:")
+    ht.describe()
+
+    logger.info("Final variant count: %d", ht.count())
+
+
+if __name__ == "__main__":
```

Comment:
add data_type parameter to the other script (and have it renamed without "exomes" in it